city_scrapers/spiders/chi_standards_tests.py

```diff
@@ -0,0 +1,82 @@
+import datetime
+
+from city_scrapers_core.constants import COMMITTEE
+from city_scrapers_core.items import Meeting
+from city_scrapers_core.spiders import CityScrapersSpider
+
+
+class ChiStandardsTestsSpider(CityScrapersSpider):
+    name = "chi_standards_tests"
+    agency = "Chicago Committee on Standards and Tests"
+    timezone = "America/Chicago"
+    start_urls = [
+        "https://www.chicago.gov/city/en/depts/bldgs/supp_info/committee_on_standardsandtests.html"
+    ]
+
+    def parse(self, response):
+        """
+        `parse` should always `yield` Meeting items.
+        Change the `_parse_title`, `_parse_start`, etc methods to fit your scraping
+        needs.
+        """
+        for item in response.xpath("//div[@class='col-xs-12']/table[1]//td/p"):
```

Comment:
It looks like this isn't parsing years before 2019. We might not always want to pull all meetings, but we should at least pull the past two years if available and be explicit about where this is being set. It would be good to pull those if possible